tests/pyfunc/test_model_export_with_loader_module_and_data_path.py

```diff
@@ -556,6 +556,15 @@ def test_column_schema_enforcement_no_col_names():
     assert pyfunc_model.predict(d).equals(pd.DataFrame(d))
 
 
+def test_model_uuid():
+    m = Model()
+    assert m.model_uuid and len(m.model_uuid) == 32
```

Comment:
Nit: https://stackoverflow.com/a/54254115/6943581 describes how we can check if a string is a valid uuid.